app/views/partials/client/manage.blade.php

```diff
@@ -29,8 +29,8 @@
             <th>{{trans('site.name')}}</th>
             <th>{{trans('site.username')}}</th>
             <th>{{trans('site.password')}}</th>
-            <th>{{trans('site.edit')}}</th>
-            <th>{{trans('site.delete')}}</th>
+            <th width="5%"><center>{{trans('site.edit')}}</center></th>
```

Comment:
[Again](#r30899153) `center` tag is frowned upon and so is the width property here.